statsmodels/stats/stattools.py

```diff
@@ -105,7 +119,7 @@ def jarque_bera(resids, axis=0):
 
     # Calculate the Jarque-Bera test for normality
     n = resids.shape[axis]
-    jb = (n / 6.) * (skew**2 + (1 / 4.) * (kurtosis - 3)**2)
+    jb = (n / 6.) * (skew ** 2 + (1 / 4.) * (kurtosis - 3) ** 2)
```

Comment:
(I'm giving up on "no spaces around `**`", given that pep-8 enforcers seems to become stronger )